lib/components/artist/article.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View, Text } = React;
+
+import colors from '../../../data/colors';
+import SerifText from '../text/serif';
+import ImageView from '../opaque_image_view';
+
+class Article extends React.Component {
+  render() {
+    const article = this.props.article
+    return (
+      <View style={{width: 300, marginBottom: 40}}>
+        <ImageView style={{ height: 175, marginRight: 20 }}
+                     aspectRatio={1.7}
```

Comment:
Should I be using the aspect ratio of the actual image so it doesn't get stretched out?